pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -932,6 +932,8 @@ func (m *kubeGenericRuntimeManager) doBackOff(pod *v1.Pod, container *v1.Contain
 		return true, err.Error(), kubecontainer.ErrCrashLoopBackOff
 	}
 
+	// Use the finished time of the latest exited container as the start point to calculate whether to do back-off.
+	ts := cStatus.FinishedAt
 	backOff.Next(key, ts)
```

Comment:
for consistency, since `backoff.Next()` also checks expiration you should use `now` I guess